CHANGELOG

```diff
@@ -1,3 +1,33 @@
+v4.0.0.pre7 (192 commits)
+-------------------------------
+
+Improvements (5)
+----------------
+
+* Implement translate method on application [1280e92]
+* Implement arrow art element [b9cd00b]
+* Allow attaching elements via style [9896ef2]
+* Remove adjusted versions of samples [9e1ea56, 762a395, a2a9a88, ff0adff]
+* Numerous Rubocop rules now in force! Now Ruby 2.3 flavored!
```

Comment:
Imo that's nothing for the changelog (and the one above is also questionable) as it's not really user facing so far as far as I can tell :)